doc/architecture/decisions/0011-build-hcadmin-app-with-qt-c.md

```diff
@@ -0,0 +1,34 @@
+# 11. Build HCadmin app with Qt/C++
+
+Date: 2018-05-16
+
+## Status
+
+proposed
+
+## Context
+
+In the alpha version we had *hcd* that runs a single Holochain app/agent as a background demon.
+
+Going forward we need a more sophisticated container of several Holochain apps for several reasons:
+* UX
```

Comment:
One end-user friendly tool to administer all installed and running Holochain apps.